pandas/core/generic.py

```diff
@@ -4688,6 +4688,23 @@ def _where(self, cond, other=np.nan, inplace=False, axis=None, level=None,
         Returns
         -------
         wh : same type as caller
+
+        Notes
+        -----
+        Because it is a function attribute, the signature for `DataFrame.where`
+        differs from `numpy.where`. Roughly `df1.where(m, df2)` is equivalent
+        to `np.where(m, df1, df2`.
```

Comment:
Double backticks are needed in rst for code.

And the closing `)` was forgotten